packages/patternfly-3/patternfly-react/src/components/VerticalNav/VerticalNavItemHelper.js

```diff
@@ -266,7 +263,7 @@ class BaseVerticalNavItemHelper extends React.Component {
         // NOTE onItemBlur takes a boolean, we want to prevent it being passed a truthy event.
         onMouseLeave={e => this.onItemBlur(false)}
       >
-        <a href="#" onClick={this.onItemClick}>
+        <a href={href} onClick={this.onItemClick}>
```

Comment:
items not setting href may not show the link correctly, seems to be working ok though (likely styled to always have the pointer cursor)/